src/meson.build

```diff
@@ -0,0 +1,50 @@
+flatpak_builder_sources = files(
+  'builder-cache.c',
+  'builder-context.c',
+  'builder-extension.c',
+  'builder-flatpak-utils.c',
+  'builder-git.c',
+  'builder-main.c',
+  'builder-manifest.c',
+  'builder-module.c',
+  'builder-options.c',
+  'builder-post-process.c',
+  'builder-sdk-config.c',
+  'builder-source.c',
+  'builder-source-archive.c',
+  'builder-source-bzr.c',
+  'builder-source-dir.c',
+  'builder-source-extra-data.c',
+  'builder-source-file.c',
+  'builder-source-git.c',
+  'builder-source-inline.c',
+  'builder-source-patch.c',
+  'builder-source-script.c',
+  'builder-source-shell.c',
+  'builder-source-svn.c',
+  'builder-utils.c',
+)
+
+glib_req = '>= 2.44'
+flatpak_builder_deps = [
+  dependency('glib-2.0', version: glib_req),
+  dependency('gio-2.0', version: glib_req),
+  dependency('gio-unix-2.0', version: glib_req),
+  dependency('json-glib-1.0'),
+  dependency('libcurl'),
+  dependency('libelf', version: '>= 0.8.12'),
+  dependency('libglnx', default_options: ['tests=false']),
+  dependency('libsoup-2.4'),
+  dependency('libxml-2.0', version: '>= 2.4'),
+  dependency('ostree-1', version: '>= 2017.14'),
+  dependency('yaml-0.1', required: get_option('yaml')),
```

Comment:
I could point you toward autotools wrap I did for libyaml if you want to easily support a subproject for it.

Reply:
... this, on the other hand, sounds like a good idea!

Reply:
Added